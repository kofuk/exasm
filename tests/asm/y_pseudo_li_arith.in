.li r0, @foo+0x1
@foo nop
