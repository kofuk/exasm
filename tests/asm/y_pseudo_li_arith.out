@00 00110000 00000000 // lui r0, 0x00
@02 01011000 00000101 // ori r0, 0x05
@04 00000000 00000000 // nop
